torch/distributed/checkpoint/filesystem.py

```diff
@@ -130,7 +130,7 @@ def values(self) -> Iterator[tuple[torch.Tensor, object]]:
         for _, obj in self.items:
             tensor = self.resolve_fun(obj).detach()
             tensor = tensor.cpu()
-            if tensor.storage().size() != tensor.numel():
+            if tensor.untyped_storage().size() != tensor.nbytes:
```

Comment:
```suggestion
            if tensor.untyped_storage().size() != tensor.nbytes():
```
It's a function, not a property, right?